### ISSUE TITLE
Search Bar Label Not Linked to Input

### ISSUE THREAD

**Issue by rosiegrant** [yext/answers-search-ui#454]
1. Our search label references the input `name` instead of the input `id`, so it's not correctly associating the label with the input. In other words,

this is what it is today:
```
<label class="yxt-SearchBar-label" for="query">Conduct a search</label>
      <input class="js-yext-query yxt-SearchBar-input" type="text" name="query" value="" autocomplete="off" autocorrect="off" spellcheck="false">
```

what I think it should be, based off of this, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label: 
```
<label class="yxt-SearchBar-label" for="query">Conduct a search</label>
      <input class="js-yext-query yxt-SearchBar-input" type="text" value="" autocomplete="off" autocorrect="off" spellcheck="false" id="query" name="query">
```

2. It's possible a user could have multiple search bars on the page (like a geolocation filter, a few filter searches), so we might run into problems with duplicate ids and the labels getting confused. could we update the ids to be unique? (Note that someone could hypothetically add multiple of the same component, like several filtersearchers, using the `name` attribute, so we could maybe use some combo of the component name + the name attribute).

**Comment by rosiegrant** [comment]
@tmeyer2115 @oshi97 could we get this into version 13 if it's quick?

**Comment by rosiegrant** [comment]
discussed in standup, tom offered to look into this thursdqy

**Comment by oshi97** [comment]
Fixed as part of v0.13.1 with https://github.com/yext/answers/pull/571
